.github/workflows/notify.yml

```diff
@@ -0,0 +1,16 @@
+name: Notify libraries
+
+on:
+  push:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Do you need the actions ehre?

Reply:
Not really, I'll remove it and if we ever need it's easy to bring back.

Reply:
Removed!